datamgmt/load/g-register_files.html.md.erb

```diff
@@ -0,0 +1,213 @@
+ 
+The `hawq register` command can register HDFS files  by using metadata loaded from a YAML configuration file by using the `--config <yaml_config\>` option. Both AO and Parquet tables can be registered. Tables need not exist in HAWQ before being registered. This function can be useful in disaster recovery, allowing information created by the `hawq extract` command to be used to re-create HAWQ tables.
+
+You can also use a YAML confguration file to append HDFS files to an existing HAWQ table or create a table and register it into HAWQ.
+
+For disaster recovery, tables can be re-registered using the HDFS files and a YAML file. The clusters are assumed to have data periodically imported from Cluster A to Cluster B. 
+
+Data is registered according to the following conditions: 
+
+-  Existing tables have files appended to the existing HAWQ table.
+-  If a table does not exist, it is created and registered into HAWQ. The catalog table will be updated with the file size specified by the YAML file.
+-  If the --force option is used, the data in existing catalog tables is erased and re-registered. All HDFS-related catalog contents in `pg_aoseg.pg_paqseg_$relid ` are cleared. The original files on HDFS are retained.
+
+Tables using random distribution are preferred for registering into HAWQ. If hash tables are to be  registered, the distribution policy in the YAML file must match that of the table being registered into. 
+
+In registering hash tables, the size of the registered file should be identical to or a multiple of the hash table bucket number. When registering hash distributed tables using a YAML file, the order of the files in the YAML file should reflect the hash distribution.
+
+
+###Example: Registration using a YAML Configuration File
+
+This example shows how to use hawq register to register HDFS data using a YAML configuration file generated by hawq extract. 
+
+First, create a table in SQL and insert some data into it.  
+
+```
+create table paq1(a int, b varchar(10))with(appendonly=true, orientation=parquet);
+```
+
+In SQL, run:
+
+```
+insert into paq1 values(generate_series(1,1000), 'abcde');
+```
+
+Go into the hawq administration utilities, and extract the table metadata by using the `hawq extract` utility.
+
+```
+hawq extract -o paq1.yml paq1
+```
+
+Register the data into new table paq2, using the --config option to identify the YAML file.
+
+```
+hawq register --config paq1.yml paq2
+```
+In SQL, select the new table and check to verify that  the content has been registered.
+
+```
+select count(*) from paq2;
+```
+
+
+##Data Type Mapping<a id="topic1__section4"></a>
+
+HAWQ and parquet tables and HIVE and HAWQ tables use different data types. Mapping must be used for metadata compatibility. You are responsible for making sure your implementation is mapped to the appropriate data type before running `hawq register`. The tables below show equivalent data types, if available.
```

Comment:
Reword first sentence:  Hive and parquet tables use different data types than HAWQ tables.